platform/web-security/src/main/resources/META-INF/motech/securityContext.xml

```diff
@@ -151,4 +151,10 @@
         <constructor-arg value="/module/server/changepassword"/>
     </bean>
 
+    <bean id="viewResolver" class="org.springframework.web.servlet.view.InternalResourceViewResolver">
```

Comment:
what's the reason for moving this? It's not related to the security configuration